app/assets/javascripts/search.js

```diff
@@ -77,14 +77,23 @@
       })
     }
 
+    var _preventSearchFilterSubmit = function () {
```

Comment:
It might be more intention revealing, (and also cover a few more edge cases) if this binds to the `submit` event, and prevents that? I think that will achieve the same thing.